packages/voila/package.json

```
@@ -8,9 +8,9 @@
   "main": "lib/index.js",
   "browserslist": ">0.8%, not ie 11, not op_mini all, not dead",
   "dependencies": {
-    "@jupyter-widgets/base": "^4.0.0",
-    "@jupyter-widgets/controls": "^3.0.0",
-    "@jupyter-widgets/jupyterlab-manager": "^3.0.0",
+    "@jupyter-widgets/base": "^5.0.0-beta.0",
+    "@jupyter-widgets/controls": "^4.0.0-beta.0",
+    "@jupyter-widgets/jupyterlab-manager": "^4.0.0-beta.0",
```

It would be interesting to update this PR to the latest packages to see if there is more code updates to do:

```suggestion
    "@jupyter-widgets/base": "^6.0.1",
    "@jupyter-widgets/controls": "^5.0.1",
    "@jupyter-widgets/jupyterlab-manager": "^5.0.3",
```

---

Fixed in https://github.com/voila-dashboards/voila/pull/1046/commits/d2e8b547d1bedad201ee5037d443db1bc124fa74